packages/backend/src/lib/serviceConfigurationManager/getters/getModelConfigurationsMap.js

```diff
@@ -0,0 +1,22 @@
+module.exports = function getModelConfigurationsMap({ serviceConfigurations }) {
+  const modelConfigurations = {}
+  Object.keys(serviceConfigurations)
+    .filter(serviceName => {
+      return !!serviceConfigurations[serviceName]
+    })
+
```

Comment:
I think there shouldn't be a linebreak in a chain of commands. interesting that prettier allowed this